scripts/deployment-test/fly-dockerfile

```diff
@@ -0,0 +1,44 @@
+# base node image
```

Comment:
Does the create-remix fly not generate this properly for us? I wouldn't expect us to need to have this file.

Reply:
it probably does, iirc i added it when i tried to make the app on fly with the api - let me verify

Reply:
yup, it does! removing now :)